modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+
+This variable now only affects the color of the separator between the {productname} Menu bar and the {productname} Toolbar.
+
+=== Available Premium plugins, which are listed by name in the Help dialog, are no longer translated.
+
+Previously, when the *Help > Help* dialog was invoked, and the _Plugins_ list was selected, available Premium plugins were supposed to be presented in alphabetical order by name.
+
+This was done manually, however, and the manual presentation was not entirely alphabetical.
+
+Also, this manual sort did not take account of the selected UI language and the Premium plugin names were not, in the main, translated.
+
+With this update Premium plugin names are explicitly not translated. But they are, now, correctly sorted in alphabetical order by name.
+
+[[bug-fixes]]
+== Bug fixes
+
+=== The Autolink plugin did not work when text nodes in the content were fragmented.
+
+The Spell Checker Pro plugin fragments text nodes as part of its normal workings. Previously however, this fragmentation broke the URL text detection of the Autolink plugin.
+
+This presented as, for example, FQDN strings such as `www.example.com` not being converted into a link.
+
+In {productname} 6.2, the Autolink plugin is now able to handle this text node fragmentation and correctly converts URL-like strings such as FQDNs into a clickable link.
+
+=== Fixed multiple incorrect types on public APIs found while enabling TypeScript strict mode.
+
+As part of switching to TypeScript’s strict mode, multiple public {productname} APIs were found to be typed incorrectly.
+
+These types have been corrected types in {productname} 6.2. 
+
+=== The number of blank lines returned from `editor.getContent({format: 'text'})` differed between browsers.
+
+In some browsers, blank lines in a {productname} document were being returned as plain text by `editor.getContent({format: 'text'})` with a different number of newlines — `+\n+` — than expected.
+
+For example, if the underlying HTML was
+
+[source,html]
+----
+<p>line-1</p>
+<p> </p>
+<p>line-2</p>
+----
+
+`editor.getContent({format: 'text'})` could, depending on the host browser, return
+
+[listing]
+----
+line-1\n\n\n\n\nline-2
+----
+
+This is one more newline character than expected.
+
+There were two related causes for this behaviour: an attribute not being assigned to a `<br>` tag when required, and {productname} not removing unnecessary elements when parsing the content.
+
+For this update, both these causes have been corrected, and the number of newlines returned as plain text by `editor.getContent({format: 'text'})` is the number expected.
+
+=== The editor focused via the `auto_focus` option was not scrolled into the viewport.
+
+Previously, when a {productname} editor was not in view and was set to auto focus, the editor was focused but did not scroll into view.
+
+This only occurred if the host browser was Google Chrome.
```

Comment:
This happened on other browsers IIRC, it was jus the one tested on. I'd suggest either removing this or just make it more generic and say "some browsers".

Reply:
To quote the commit message:

> In *the editor focused via the `auto_focus` was not scrolled into the viewport* section, minor re-write to turn the specific (Google Chrome) in to the general (some browsers).